intellij/resources/META-INF/plugin.xml

```diff
@@ -21,7 +21,7 @@
   </change-notes>
 
   <!-- please see http://www.jetbrains.org/intellij/sdk/docs/basics/getting_started/build_number_ranges.html for description -->
-  <idea-version since-build="141.0"/>
+  <idea-version since-build="162"/>
```

Comment:
Actually I though it was the version it's been built with.
I will revert that then.